aries-site/src/pages/templates/reverse-navigation.mdx

```diff
@@ -1,3 +1,36 @@
-import { ComingSoon } from '../../components';
+import { Image } from 'grommet';
+import { DiagramPreview, Example } from '../../layouts';
+import {
+  NameValueListFontWeightSecondPreview,
+} from '../../examples';
 
-<ComingSoon />
\ No newline at end of file
+## When to Use:
+
+- When the user may want to return to the previous page’s content
```

Comment:
I don't think this is an accurate description of the use of this pattern. I thought we were leaning more towards the term "Context" or "Up" to indicate that the user would be taken to the page containing this page in the overall site hierarchy. If the user did not navigate to this page from that containing page, then clicking the button would not return to the "previous" page, nor is it aligned to anything "chronological".

Reply:
@vavalos5 @britt6612 @ericsoderberghp 

I am a bit confused on which pattern we are presenting here.

Are we presenting:
1) a pattern which allows a user to step back through their history?
OR
2) a pattern which allows a user to navigate from child to parent / from spoke to its hub?


Clarification on this point will help me review.